sympy/functions/elementary/complexes.py

```diff
@@ -1000,6 +1181,15 @@ class principal_branch(Function):
     >>> principal_branch(exp_polar(2*pi*I)*3*z, 2*pi)
     3*principal_branch(z, 2*pi)
 
+    Parameters
```

Comment:
I'd also reformat the above examples sections with `Examples` tag

Reply:
Done.